Package@swift-5.9.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version: 5.6
+// swift-tools-version: 5.8
```

Comment:
```suggestion
// swift-tools-version: 5.9
```

Reply:
We haven't dropped 5.8 support yet. So I set the lowest version